### ISSUE TITLE
you might want to do a version (tag) 1.0.0?

### ISSUE THREAD

**Issue by tubssp** [chanmix51/SlapOM#17]
you might want to do a version (tag) 1.0.0?

**Comment by chanmix51** [comment]
Yes. I want to release Pomm 2.0.0 first then, I will take time to have a serious look at SlapOM.

**Comment by tubssp** [comment]
ping @chanmix51

**Comment by chanmix51** [comment]
Lot of work to be done on SlapOM. I will take time to organize the 1.0 milestone.

**Comment by chanmix51** [comment]
I might find time to give SlapOM some attention. The biggest problem for now and why it cant be taged as stable is : there are no unit tests.

Why aren't there any tests ? Because I do not know how to setup a LDAP server somewhere and perform read / write tests.
